src/app/shared/components/timer/timer.component.html

```diff
@@ -0,0 +1,10 @@
+<p>This thing work</p>
+
+<div class="timer">
+  <span class="time">{{ time.length < 3 ? 0 : (time | slice : 0 : -2) }}</span>
+  <span class="time">{{ time | slice : -2 }}</span>
```

Comment:
to keep same logic you can use [date](https://angular.io/api/common/DatePipe) and [number](https://angular.io/api/common/DecimalPipe) pipes instead of transforming string to number all time
  <span class="time">{{ time / 100 | number: '1.0-0'  }}</span>
  <span class="time">{{ time * 10 | date : 'SS' }}</span>

Reply:
Interesting fixed.